bucket/wechat.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": "3.7.5.23",
+    "description": "Free messaging and calling app by Tencent",
+    "homepage": "https://pc.weixin.qq.com/",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://www.wechat.com/en/service_terms.html"
+    },
+    "notes": "We don't persist your WeChat data storaged in \"$env:APPDATA\\Tencent\\WeChat\".",
```

Comment:
```suggestion
```
I don't think this is needed. Many apps in *Scoop* stores their data in `$Env:AppData`, but we don't show it explicitly.